GVRf/Framework/src/org/gearvrf/GVRMaterial.java

```diff
@@ -70,7 +70,10 @@
     private int mShaderFeatureSet;
     private GVRMaterialShaderId shaderId;
     final private Map<String, GVRTexture> textures = new HashMap<String, GVRTexture>();
+    // ---------------------------------------------AR------------------------{
+    public static float ar_offsets[] = new float[4];
 
```

Comment:
Naming should follow camel naming in Java. Thanks.

Also I am not considering making it public is a good practice for this parameter.